### ISSUE TITLE
the power of enemy

### ISSUE THREAD

**Issue by Fenix-sudo** [thetrueoneshots/PyroProgression#25]
the damage and haste of the enemy is pretty high compared to the stat we can obtain in the game even with gear highter the monster can easily kill you in no time

**Comment by pervisbr** [comment]
> the damage and haste of the enemy is pretty high compared to the stat we can obtain in the game even with gear highter the monster can easily kill you in no time

True... I downloaded the mod to test and found it very hard to level up because white and green monsters can deal 190+ damage in one swing...
To be able to level up in a easy way you have to wait for the skills to cool down and spam it,, unless it's a group of archers and mages, then you have to ignore them even though they are white/green.
